### PR TITLE
Implemented functionality for merge_sorted_lists function and added t…

### PR DESCRIPTION
**Pull request by kimcodes3**
…ests. All tests have passed.

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
+
```
Comment (hamsurtado) [inline functions/merge_sorted_lists.py:3]: excellent work

File: functions/merge_sorted_lists.py [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+    return sorted(merged)
```
Comment (red-med) [inline functions/merge_sorted_lists.py:4]: genius - well done!

File: tests/merge_sorted_lists_test.py [hunk 3]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
+
+def test_for_1_and_2():
```
Comment (hamsurtado) [inline tests/merge_sorted_lists_test.py:5]: good use of snake case

File: tests/merge_sorted_lists_test.py [hunk 4]
```diff
@@ -1,2 +1,38 @@
...
+def test_for_1_2_3_and_4_5_6():
+    # Arrange
+    l1 = [1, 2, 3]
+    l2 = [4, 5, 6]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2, 3, 4, 5, 6]
```
Comment (red-med) [inline tests/merge_sorted_lists_test.py:38]: tests are not that robust... test number 3 is already sorted so may want to try with the input lists reversed.

**Review by hamsurtado — COMMENTED** [review]
code looks excellent, please see emojis for additional explanation

**Review by red-med — CHANGES_REQUESTED** [review]
some tweaks to tests should be all we need!

**Comment by kimcodes3** [comment]
Very Cool

**Pull request merged**